library/Haste/EventListener/TemplateHelperListener.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * Haste utilities for Contao Open Source CMS
+ *
+ * Copyright (C) 2012-2013 Codefog & terminal42 gmbh
+ *
+ * @package    Haste
+ * @link       http://github.com/codefog/contao-haste/
+ * @license    http://opensource.org/licenses/lgpl-3.0.html LGPL
+ */
+
+namespace Haste\EventListener;
+
+class TemplateHelperListener
+{
+    /**
+     * @param \Template $template
+     */
+    public function onParseTemplate(\Template $template)
+    {
+        $template->hasteTemplateHelpers = new TemplateHelpers();
```

Comment:
I would just make it `$template->haste` for the sake of simplicity. Imo it's obvious those are the template helpers and since the Haste is rather an own name nobody will override it anyway.